content/docs/04-clusters/01-public-cloud/01-aws.md

```diff
@@ -946,6 +963,33 @@ To create an AWS cloud account using STS credentials follow the steps below:
 </Tabs.TabPane>
 </Tabs>
 
+## Create AWS US Gov Cloud Account

```

Comment:
@resmaa this is a how to so we need to follow the How-to [layout guide](https://spectrocloud.atlassian.net/wiki/spaces/DE/pages/1774944263/How-to+Layout). This means we need to add prerequisites, the enablement steps, and a validation section.